src/LinAlg/hiopVectorRajaImpl.hpp

```diff
@@ -2024,14 +2073,11 @@ bool hiopVectorRajaPar::isfinite_local() const
  * 
  * @pre Vector data was moved from the memory space to the host mirror.
  */
-void hiopVectorRajaPar::print(FILE* file/*=nullptr*/, const char* msg/*=nullptr*/, int max_elems/*=-1*/, int rank/*=-1*/) const
+template<class MEM, class POL>
+void hiopVectorRaja<MEM,POL>::
+print(FILE* file, const char* msg/*=NULL*/, int max_elems/*=-1*/, int rank/*=-1*/) const
 {
-  int myrank=0, numranks=1;
-
-  if(nullptr == file) {
-    file = stdout;
-  }
```

Comment:
I also think keeping this logic could be helpful, if not just defaulting the `file` variable to `stdout`.

Reply:
fixed